packages/flutter/lib/src/foundation/_isolates_io.dart

```diff
@@ -80,7 +80,7 @@ class _IsolateConfiguration<Q, R> {
 Future<void> _spawn<Q, R>(_IsolateConfiguration<Q, FutureOr<R>> configuration) async {
   R result;
   await Timeline.timeSync(
-    '${configuration.debugLabel}',
+    '${configuration.debugLabel}', // ignore: unnecessary_string_interpolations, guarding null value
```

Comment:
The only ignore case

Reply:
We could also just change this to coallesce to something like `<isolate label optimized out>`.

Reply:
I will rebase my PR once #49638 is merged.